pkg/controller/pxc/users.go

```diff
@@ -275,12 +264,25 @@ func (r *ReconcilePerconaXtraDBCluster) restartProxy(cr *api.PerconaXtraDBCluste
 	return nil
 }
 
-// getInternalSysUsersSecret return secret created by operator for storing system users data
-func (r *ReconcilePerconaXtraDBCluster) getInternalSysUsersSecret(cr *api.PerconaXtraDBCluster, sysUsersSecretObj *corev1.Secret) (corev1.Secret, error) {
-	secretName := internalPrefix + cr.Spec.SecretsName
-	internalSysUsersSecretObj, err := r.getInternalSysUsersSecretObj(cr, sysUsersSecretObj)
+// getSysUsersSecrets return internal and external secrets for storing system users data
+func (r *ReconcilePerconaXtraDBCluster) getSysUsersSecrets(cr *api.PerconaXtraDBCluster) (internalSysUsersSecretObj, sysUsersSecretObj corev1.Secret, err error) {
```

Comment:
should we use named returns?
We are always specifying return arguments and even creating them(like `return corev1.Secret{}.....`)